asciidoctorj-core/src/main/resources/org/asciidoctor/internal/asciidoctorclass.rb

```diff
@@ -19,50 +21,54 @@ def unregister_all_extensions()
         Asciidoctor::Extensions.unregister_all
     end
 
-    def docinfo_processor(extensionName)
-        Asciidoctor::Extensions.register do
+    def unregister_extension name
+        Asciidoctor::Extensions.unregister name
+    end
+
+    def docinfo_processor(extensionName, groupName = nil)
+        Asciidoctor::Extensions.register (groupName != nil ? groupName.to_sym : nil) do
```

Comment:
Don't put a space before the opening bracket. That causes a warning in some versions of Ruby.

You no longer need to coerce to a symbol as Asciidoctor 1.5.6.1 does that for you. Just pass the groupName as is.